testsuite/features/secondary/srv_push_package.feature

```diff
@@ -11,6 +11,21 @@ Feature: Push a package with unset vendor
   Scenario: Log in as admin user
     Given I am authorized for the "Admin" section
 
+  @uyuni
+  Scenario: Pre-requisite: SLES minion must be subscribed to the openSUSE Leap Micro 5.5 channel
+    Given I am on the Systems overview page of this "sle_minion"
+    When I follow "Software" in the content area
+    And I follow "Software Channels" in the content area
+    And I wait until I do not see "Loading..." text
+    And I check radio button "openSUSE Leap Micro 5.5 (x86_64)"
+    And I check "Uyuni Client Tools for openSUSE Leap Micro 5.5 (x86_64)"
+    And I click on "Next"
+    Then I should see a "Confirm Software Channel Change" text
+    When I click on "Confirm"
+    Then I should see a "Changing the channels has been scheduled." text
+    When I follow "scheduled" in the content area
+    And I wait until I see "1 system successfully completed this action." text, refreshing the page
```

Comment:
This line will fail if we try to run this test in parallel with others.
So... can you please move this test from 

https://github.com/uyuni-project/uyuni/blob/master/testsuite/run_sets/secondary_parallelizable.yml#L37

To the other secondary.yaml file, or find a way to work in parallel?

Reply:
Thank you

Reply:
I took one of scenarios which does the same but with different channels and is also parallelizable i.e https://github.com/uyuni-project/uyuni/blob/09c6af7be4525f1d1edb8fc584b83d232c541378/testsuite/run_sets/secondary_parallelizable.yml#L61

So following:
https://github.com/uyuni-project/uyuni/blob/09c6af7be4525f1d1edb8fc584b83d232c541378/testsuite/features/secondary/min_bootstrap_api.feature#L68

I adapted the last parts of to instead of following scheduled area and checking if system completed the action, I just wait until the event is completed. What do you think?

Reply:
I think that part might still failing if we run in parallel with other tests : 
"1 system successfully completed this action."

But go ahead and merge, it's fine, we will worry about parallelizable part in a different PR, when we test again this secondary_parallelizable.yaml running all tests in parallel.

Reply:
Can you refresh the diff? I recently removed this part already.

Reply:
Sorry, my bad, I was looking into _Then I should see a "1 package install has been scheduled for" text_ but that step will not affect on parallel tests.
All good! Thank you :)

Reply:
Ok, resolving.